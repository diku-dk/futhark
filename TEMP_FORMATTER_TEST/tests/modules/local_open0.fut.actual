82d938af0c647295527087e4dcde2af7
