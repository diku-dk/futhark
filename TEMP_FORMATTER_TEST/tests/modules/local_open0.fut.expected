efc3c1f3fc310b3c0cafeec1706d71ce
